src/ir_Hitachi.cpp

```diff
@@ -551,12 +551,11 @@ void IRHitachiAc424::stateReset(void) {
   setPower(true);
   setMode(kHitachiAc424Cool);
   setFan(kHitachiAc424FanAuto);
-  setInvertedStates();
 }
 
 void IRHitachiAc424::setInvertedStates(void) {
-  for (uint8_t i = 3; i < kHitachiAc424StateLength; i+=2)
-    remote_state[i+1] = ~remote_state[i];
+  for (uint8_t i = 3; i < kHitachiAc424StateLength - 1; i += 2)
```

Comment:
Good catch, woops 😬

Reply:
Yeah. I should have caught it in review.